### DISCUSSION TITLE
Text blur problem while using realbasicvsr

### DISCUSSION THREAD

**Discussion by mayunjie** [open-mmlab/mmediting#1228] [Models]
![图片](https://user-images.githubusercontent.com/23325496/186560125-9d51bf7a-b4fb-4348-b3cb-68b374493a7c.png)

Apply realbasicvsr to video with text alway generate the text blur problem.
We have try to select the specified dataset,but the problem still occured.
What leads to this problem and how to fix tis problem

**Comment by zengyh1900** [comment]
Please check this issue. @Z-Fran

**Comment by zhu2bowen** [comment]
@mayunjie 有解决方案吗？Any progress？
